PowerEditor/src/ScitillaComponent/AutoCompletion.cpp

```diff
@@ -43,11 +43,7 @@ static bool isInList(generic_string word, const vector<generic_string> & wordArr
 
 static bool isAllDigits(const generic_string &str)
 {
-#ifdef _UNICODE
 	return std::all_of(str.begin(), str.end(), ::iswdigit);

```

Comment:
`_istdigit` if you want to care about both. Since `generic_string` is typedef'd, I'd advice using a TCHAR version.

Reply:
Thank you for Code Review. I will fix that on next commit.

but I have one question, generic_string is templated string? For Example. std::wstring same as generic_string<WCHAR>?

Reply:
Yep. From `Common.h`:
```cpp
typedef std::basic_string<TCHAR> generic_string;
typedef std::basic_stringstream<TCHAR> generic_stringstream;
```

Reply:
Thank You For Answer. I fix that.